README.md

```diff
@@ -11,22 +11,23 @@
 ## Description
 
 ### Intended Use
-This validator uses BespokeAI's factcheck API to evaluate the generated text against the provided context. It checks if the claims in the generated text are supported by the given context, using a configurable threshold for determining support. The validator can process the input as a whole or split it into individual sentences for more granular evaluation.
+This validator uses BespokeLabs.AI's minicheck API to evaluate LLM generated text against the provided context. It checks if the claims in the generated text are supported by the given context, using a configurable threshold for determining support. The validator can process the input as a whole or split it into individual sentences for more granular evaluation.
 
 ### Requirements
 
 * Dependencies:
 	- guardrails-ai>=0.4.0
 	- nltk
-	- requests
+	- bespokelabs
+	- tenacity
```

Comment:
why do we need this?

Reply:
this == tenacity dependency

Reply:
It is used to retry etc.
`from tenacity import retry, stop_after_attempt, wait_exponential`